05-JWT-Basics/starter/errors/custom-error.js

```diff
@@ -1,8 +1,7 @@
 class CustomAPIError extends Error {
-  constructor(message, statusCode) {
-    super(message)
-    this.statusCode = statusCode
```

Comment:
Another option is to still let this have a status code, and then do super(message, <your status code>) in the error classes that extend it. Not required, just informational